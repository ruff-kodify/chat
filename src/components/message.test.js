import React from 'react';
import Message from './message';

test('it should render properly', () => {
  const wrapper = shallow(
    <Message
      id={ 1 }
      type="user"
      body="hello world!"
    />
  );
  expect(wrapper).toMatchSnapshot();
});
