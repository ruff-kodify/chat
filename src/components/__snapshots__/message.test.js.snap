// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render properly 1`] = `
<div
  className="message message--user"
  data-id={1}
>
  hello world!
</div>
`;
